packages/mui-material/src/Alert/alertClasses.ts

```diff
@@ -10,29 +10,85 @@ export interface AlertClasses {
   outlined: string;
   /** Styles applied to the root element if `variant="standard"`. */
   standard: string;
-  /** Styles applied to the root element if `variant="standard"` and `color="success"`. */
+  /** Styles applied to the root element if `color="success"`. */
+  colorSuccess: string;
```

Comment:
Wasn't the decision made not to add default class names to the HTML to avoid bloating the HTML structure? You can target them with `root`. I see the need for it now since we already have it for composed classes. But I assume these will be removed when deprecations are also removed, right?

Reply:
We discussed this one final time and decided that **default value classes will be added:** https://github.com/mui/material-ui/issues/40733

The reasons were:
- Provide the same API for all prop values, avoiding issues when changing the default via `defaultProps`
- Provide the same specificity for all prop CSS classes
- By removing the composed classes, we reduce the bloat significantly, so it shouldn't be an issue